src/analysis/seminorm.lean

```diff
@@ -392,6 +392,24 @@ begin
   { rw [finset.sup'_cons hs, finset.inf'_cons hs, ball_sup, inf_eq_inter, ih] },
 end
 
+lemma ball_mono {p : seminorm 𝕜 E} {r₁ r₂ : ℝ} (h : r₁ ≤ r₂) : p.ball x r₁ ⊆ p.ball x r₂ :=
+λ _ (hx : _ < _), hx.trans_le h
+
+lemma ball_antimono {p q : seminorm 𝕜 E} (h : q ≤ p) : p.ball x r ⊆ q.ball x r :=
+λ _, (h _).trans_lt
+
+lemma add_ball_zero (p : seminorm 𝕜 E) (r : ℝ) :
+  ball p (0 : E) r + ball p (0 : E) r ⊆ ball p (0 : E) (2 * r) :=
```

Comment:
This is true as
```suggestion
  ball p x r + ball p x r ⊆ ball p x (r + r) :=
```
Right?

Reply:
not quite, but I have now the general case.